docs/testing/troubleshoot-manifest.md

```diff
@@ -63,6 +63,35 @@ You can use runtime logging to debug your add-in's manifest as well as several i
 > [!IMPORTANT]
 > Runtime Logging affects performance. Turn it on only when you need to debug issues with your add-in manifest.
 
+### Use runtime logging from the command line
+
+Enabling runtime logging from the command line is the fastest way to use this logging tool. The below commands use npx, which is provided by default as part of npm@5.2.0+. If you have an earlier version of [npm](https://www.npmjs.com/), try [Runtime logging on Windows](#runtime-logging-on-windows) or [Runtime logging on Mac](#runtime-logging-on-mac) instructions, or [install npx](https://www.npmjs.com/package/npx).
+
+To enable runtime logging, run the following:
+    ```command&nbsp;line
+	npx office-addin-dev-settings runtime-log --enable
+	```
+
+To enable runtime logging only for a specific file, use the same command with a filename. Use your own file name in the following sample, instead of filename.txt:
```

Comment:
I think the syntax for command line parameters is square brackets ([FileName]). That'll save you the line about replacing the filename with their own.